### PR TITLE
adding the field descriptions

### PR DESCRIPTION
**Pull request by sreejithgs**
Added the field descriptions for the lsstream conf file.

### CODE REVIEW

File: field-descriptions.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    The default value is 64.
+
+-  `FileSizeMax`:
+
+    For Kafka endpoint, AVRO files are created and stored on the disk before they get pushed to the endpoint.
+    This parameter specifies the size of each such file in Kibibyte (KiB). A file can contain multiple transaction records. The default value is 48 KiB.
+
+-  `RecordType`:
+
+    Specifies the types of records you want to export:
```
Comment (ritjaintibco) [inline field-descriptions.md:54]: ah can you please change this to :-  (i think this somehow got leaked)

`
HTTP
TCP
SWG
VPN
ICA
APPFW
BOT
VIDEOPT
BURST_CQA
SLA
MONGO
MQTT
`

Basically, remove `BOT_LE BOT_KM AS_FDI NGN`

File: field-descriptions.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    You must set these fields appropriately to export the required records.
+    Examples :
+
+        "TCP": "all",
+        "SWG": "none",
+        "APPFW": "all"
+
+-  `EVENTS`:
+
+    Citrix ADC Observability Exporter allows exporting time series, events, and audit logs to Splunk.
```
Comment (ritjaintibco) [inline field-descriptions.md:84]: `time series, events, and audit logs` this line can cause confusion. `timeseries` include metrics/events/auditlogs, of which only events and auditlogs are exported to splunk. can you please rephrase this?

File: field-descriptions.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+-  `transRateLimitWindow`:
+
+    Specifies the recalculation window in seconds and the value must be greater than zero.
+    The lower the window size, the more effective is the rate-limiting, but specifying low values may cause slight CPU overhead.
+    The default value is 5 seconds.
+
+-  `AuthToken`:
+
+    You can use the auth token to perform the token-based authentication for Splunk.
+    It can also be used for password-based authentication for ElasticSearch.
```
Comment (ritjaintibco) [inline field-descriptions.md:126]: Not required right now- but please add a link for this later.

**Review by ritjaintibco — CHANGES_REQUESTED** [review]

File: field-descriptions.md (hunk 1 above)
Comment (sreejithgs) [reply field-descriptions.md]: Removed.

File: field-descriptions.md (hunk 2 above)
Comment (sreejithgs) [reply field-descriptions.md]: Rewrote to allows exporting time series ( events, and audit logs) to Splunk.

**Pull request closed without merging**